components/BudgetBar/BudgetBar.module.css

```diff
@@ -1,18 +1,78 @@
-@media (min-width: 373px) {
+.basicBar {
+  background-color: #f7ecde;
+  border: solid 1px black;
+  border-radius: 20px;
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+  display: flex;
+  flex-direction: row;
```

Comment:
flex-direction row is default and can be removed

Reply:
Thank you! Removed the code!